### PR TITLE
Visitor can view articles by category

### PR DESCRIPTION
**Pull request by kaylawoodbury**
# [PT Story](https://www.pivotaltracker.com/story/show/17139723)
## User Story
```
As a visitor
In order to find an article I want to read
I would like to be able to view articles by category 
```
## Changes proposed in this pull request:
* Adds ability for user to see category options
* Adds filter on display logic for articles into the category
## What I have learned working on this feature:
* How to have two components use the same third component (display all and display by category both use display single article)
* How to update from component states to using redux
* How not to apply Redux
## Packages/Gems added
- React-Router-Dom
## Screenshots (Only if client side story)
Screenshot 2020-03-23 at 10 02 22
Screenshot 2020-03-23 at 10 03 41

## Lo-Fi (Only if client side story)
Pasted Image at 2020_03_17_04_25 pm

### CODE REVIEW

File: cypress/integration/visitorCanChooseArticleByCategory.feature.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+describe("Visitor can choose article by categort", () => {
```
Comment (pgaunitz) [inline cypress/integration/visitorCanChooseArticleByCategory.feature.js]: fix typo

**Review by pgaunitz — APPROVED** [review]
LGTM

File: cypress/integration/visitorCanChooseArticleByCategory.feature.js (hunk 1 above)
Comment (kaylawoodbury) [reply cypress/integration/visitorCanChooseArticleByCategory.feature.js]: Updated typo

**Comment by pierre-1** [comment]
LGTM. Great work and yes we will revisit the styling.

**Pull request merged**